doc/jekyll_tools/fonts/font-mfizz/font-mfizz.css

```diff
@@ -0,0 +1,318 @@
+/*
```

Comment:
These font files can be removed from here and from the CSS; these were used earlier on the homepage but not any more. (Actually, they should probably also be removed from the homepage).

Reply:
ok